src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexBoyerMoore.cs

```diff
@@ -18,29 +17,25 @@ namespace System.Text.RegularExpressions
 {
     internal sealed class RegexBoyerMoore
     {
-        internal int[] _positive;
-        internal int[] _negativeASCII;
-        internal int[][] _negativeUnicode;
-        internal String _pattern;
-        internal int _lowASCII;
-        internal int _highASCII;
```

Comment:
The following fields *are* used by `RegexCompiler` in the full framework:
 - `_positive`
 - `_negativeASCII`
 - `_negativeUnicode`
 - `_pattern`
 - `_lowASCII`
 - `_highASCII`

This class is internal; it might be better if these were exposed as public properties in the desktop code, to be more explicit about their use by another class.

The fields never change, so `readonly` still applies.

If we need to change these back to internal, we need a way to prevent someone from proposing the same changes again. Is a comment enough to do that?

Or acknowledge that since there is no `RegexCompiler` in this particular incarnation of the code, the fields should be `private` because they aren't used anywhere else in **this** code base (in this repository).